app/handlers/createHighlightVideo.py

```diff
@@ -1,8 +1,41 @@
 #!/usr/bin/env python
 
+import os
 import tornado.web
+import tornado.escape
+from trafficcloud.video import create_highlight_video, get_framerate
+from storage import alterInteractionsWithRoadUserType, getNearMissFrames
+from trafficcloud.app_config import get_project_path, get_project_video_path
```

Comment:
app_config.py has since been moved to traffic_cloud_utils. This should now reference the new location, as the old one may become unsupported.